flake.nix

```diff
@@ -20,6 +20,14 @@
 
         packages = pkgs;
 
+        # In Nix 2.8 you can run `nix fmt` to format this whole repo. Note that you need to have loaded the
+        # `nix develop` shell before so the various formatters are available in the PATH.
```

Comment:
that's true only if the dependencies are not specified in nix. if the config has references to packages then it should work even outside of `nix develop`